.github/ISSUE_TEMPLATE/issue.yaml

```diff
@@ -0,0 +1,91 @@
+name: New Issue
```

Comment:
Then this issue type would most likely be `Bug Report`, or something similar…

Reply:
Should now be resolved in the latest commit.  I also changed the file name and description to be more in line with a bug report.